styles.js

```diff
@@ -1,14 +1,229 @@
 import { createGlobalStyle } from "styled-components";
 
 export default createGlobalStyle`
-  *,
-  *::before,
-  *::after {
+:root {
+  --color-background: #f0f5f1; 
+  --color-text-primary: #2e4d36; 
+
+  --color-background-cards: #ffffff; 
+
+  --color-button-text: #ffffff; 
```

Comment:
Maybe this would be better as a single colour, with a more general name ? --white ?

Reply:
this is also on purpose (for now), incase we want to change the color specifically for buttons